sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/aggregator/DoubleLastValueAggregator.java

```diff
@@ -28,33 +32,37 @@
  * at any time.
  */
 @ThreadSafe
-final class DoubleLastValueAggregator extends AbstractAggregator<Double> {
+final class DoubleLastValueAggregator extends AbstractAggregator<DoubleAccumulation> {
+  private final Supplier<ExemplarReservoir> reservoirSupplier;
+
   DoubleLastValueAggregator(
       Resource resource,
       InstrumentationLibraryInfo instrumentationLibraryInfo,
-      MetricDescriptor descriptor) {
+      MetricDescriptor descriptor,
+      Supplier<ExemplarReservoir> reservoirSupplier) {
     super(resource, instrumentationLibraryInfo, descriptor, /* stateful= */ true);
+    this.reservoirSupplier = reservoirSupplier;
   }
 
   @Override
-  public AggregatorHandle<Double> createHandle() {
-    return new Handle();
+  public AggregatorHandle<DoubleAccumulation> createHandle() {
+    return new Handle(reservoirSupplier.get());
   }
 
   @Override
-  public Double accumulateDouble(double value) {
-    return value;
+  public DoubleAccumulation accumulateDouble(double value) {
+    return DoubleAccumulation.create(value);
   }
 
   @Override
-  public Double merge(Double a1, Double a2) {
+  public DoubleAccumulation merge(DoubleAccumulation a1, DoubleAccumulation a2) {
     // TODO: Define the order between accumulation.
```

Comment:
lol.

Reply:
sorry...just a little punchy today, and after seeing 2 different merge choice, I saw this. :)